frontend/src/constants/countries.js

```diff
@@ -0,0 +1,255 @@
+// This file contains all the countries and their 2-letter country codes. 
```

Comment:
This PR depends on this file and it's really no use alone, so I think I'll leave it here.

Reply:
In that case, you could make a separate PR that this PR relies upon. The PR with this file in it would just be submitted first. I am indifferent in this particular case, though.